lib/core/askm_app.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:askm/generated/l10n.dart';
+import 'package:askm/presentation/pages/main_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+class AskmApp extends StatefulWidget {
+  const AskmApp({super.key});
+
+  @override
+  State<AskmApp> createState() => _AskmAppState();
+}
+
+class _AskmAppState extends State<AskmApp> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      localizationsDelegates: const [
+        S.delegate,
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        GlobalCupertinoLocalizations.delegate,
+      ],
+      supportedLocales: S.delegate.supportedLocales,
+      theme: ThemeData.dark(),
+      home: const MainScreen(),
+      builder: (context, widget) => SafeArea(child: widget ?? const SizedBox()),
```

Comment:
wrapped application with SafeArea its bad practise 

because it can interfere with you on some screens, write an AppWrapper widget, make it accept a child - the child will be your layout, and in it use SafeArea and default padding on the sides, and just on some screens where this will not be needed - then don't use

<img width="632" alt="image" src="https://github.com/benikgalstyan/askm/assets/105119340/5d8edc8e-3788-4dc6-b637-9aba2cef074a">